content/it/cloud_native_tech.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Cloud Native Technology
+status: Completed
+category: Concept
+---
+
+## Cos'è
+
+Le tecnologie _cloud native_, cui ci si riferisce anche con il termine _cloud native stack_, sono le tecnologie utilizzate per la creazione di [applicazioni _cloud native_](/cloud_native_apps/). Consentendo alle organizzationi di implementare e gestire applicazioni scalabili in contesti moderni e dinamici quali le piattaforme cloud (pubbliche, private o ibride), mantengono la "promessa del cloud" e sfruttano al massimo i benefici del [_cloud computing_](/cloud_computing/). Sono progettate da zero con l'intento di impiegare le funzionalità di cloud computing e [container](/container/). Il [mesh di servizi](/service_mesh/), i [microservizi](/microservices/) e le [infrastrutture immutabili](/immutable_infrastructure) sono esempi di questo approccio. 
+
+## Quale problema affronta 
+
+Il _cloud native stack_ è formato da molte categorie di tecnologie diverse tra loro; ciascuna si propone di risolvere un problema specifico. Consultando il [CNCF Cloud Native Landscape](https://landscape.cncf.io/), potrai renderti conto di quante aree differenti coinvolga. In linea generale, il _cloud native stack_ affronta un gruppo principale di problemi, ossia gli aspetti negativi dei modelli operativi IT tradizionali. Fra gli altri vi sono: la difficoltà di creare applicazioni scalabili, _fault-tolerant_ (tolleranti gli errori) e [_self-healing_](/self_healing) (auto riparanti), o anche l'utilizzo inefficiente delle risorse.
+
+## In che modo aiuta
+
+Sebbene ogni tecnologia risponda ad un bisogno specifico, collettivamente le tecnologie _cloud native_ consentono sistemi disaccoppiati che sono insieme _resilient_(affidabili), gestibili ed osservabili. Queste caratteristiche, combinate ad un'automazione stabile, consentono agli sviluppatori di implementare cambiamenti ad alto impatto frequentemente e con esito certo a fronte di uno sforzo minimo. In sostanza, gli aspetti peculiari dei sistemi _cloud native_ si ottengono più facilmente con l'utilizzo del _cloud native stack_.
```

Comment:
- (affidabili) > non possiamo usare resilienti? ha un altro significato in questo contesto?
- gli aspetti peculiari dei sistemi _cloud native_ si ottengono più facilmente con l'utilizzo del _cloud native stack_ > gli aspetti non si "ottengono" secondo me. magari si valorizzano? si potenziano?